packages/components/src/Auth.tsx

```diff
@@ -330,8 +330,8 @@ function LoginForm() {
         <form className="space-y-6">
           <div>
             <MnemonicInput mnemonic={mnemonic} setMnemonic={setMnemonic} />
-            {!chain && <ChainInput chain={chain} setChain={setChain} />}
-            {!network && <NetworkInput network={network} setNetwork={setNetwork} />}
+            {<ChainInput chain={chain} setChain={setChain} />}
```

Comment:
With this change, all apps prompt the user for a chain and network. The way it is supposed to work that if an operator sets the chain in the `.env` file, the sign up modal does not ask for a chain. Only if the operator does not specify a chain is the user asked to provide one in the sign up modal. The same goes for network

Also, the network should be selected only in the sign up window and not in the menu bar.